include/aos/sm/networkmanager.hpp

```diff
@@ -28,6 +32,56 @@ static constexpr auto cMaxNumAliases = AOS_CONFIG_NETWORKMANAGER_MAX_NUM_ALIASES
  */
 static constexpr auto cMaxNumExposedPorts = AOS_CONFIG_NETWORKMANAGER_MAX_NUM_EXPOSED_PORTS;
 
+/**
+ * Max number of hosts.
+ */
+static constexpr auto cResolvConfLineLen = AOS_CONFIG_NETWORKMANAGER_RESOLV_CONF_LINE_LEN;
+
+/**
+ * Max exposed port len.
+ */
+static constexpr auto cExposedPortLen = cPortLen + cProtocolNameLen;
+
+/**
+ * Max CNI version length.
+ */
+constexpr auto cCNIVersionLen = AOS_CONFIG_CNI_VERSION_LEN;
```

Comment:
Keeping CNI consts and structs under separate header and separate namespace inside SM was good idea. Could you move them back to separate file?

Reply:
Done